_includes/breadcrumbs.html

```diff
@@ -5,8 +5,7 @@
 page's location in the TOC (_data/toc.yaml). To get the "parent" TOC entries
 for the current page, we:
 
-    - iterate through each of the main sections / categories in the TOC (home, guides,
-      product manuals, reference, samples)
+    - iterate through each of the main sections / categories in the TOC (home, guides, manuals, reference, samples)
```

Comment:
nit: looks like this was "un-wrapped"
```suggestion
    - iterate through each of the main sections / categories in the TOC (home, guides,
      manuals, reference, samples)
```